lib/active_model_serializers/railtie.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails/railtie'
+require 'action_controller'
+require 'action_controller/railtie'
+require 'action_controller/serialization'
+
+module ActiveModelSerializers
+  class Railtie < Rails::Railtie
+    config.to_prepare do
+      ActiveModel::Serializer.serializers_cache.clear
+    end
+
+    initializer 'active_model_serializers.action_controller' do
+      ActiveSupport.on_load(:action_controller) do
+        include(::ActionController::Serialization)
+      end
+    end
+
+    # This hook is run after the action_controller railtie has set the configuration
+    # based on the *environment* configuration and before any config/initializers are run
+    # and also before eager_loading (if enabled).
+    initializer 'active_model_serializers.set_configs', :after => 'action_controller.set_configs' do
+      ActiveModelSerializers.logger = Rails.configuration.action_controller.logger
+      ActiveModelSerializers.config.cache_store     = Rails.configuration.action_controller.cache_store
+      ActiveModelSerializers.config.perform_caching = Rails.configuration.action_controller.perform_caching
```

Comment:
I think myvcomment about loaf order is still true given tests pass

B mobile phone

> On Jan 17, 2016, at 1:37 PM, Dave Gynn <notifications@github.com> wrote:
> 
> In lib/active_model_serializers/railtie.rb:
> 
> > +      ActiveModel::Serializer.serializers_cache.clear
> > +    end
> > +
> > +    initializer 'active_model_serializers.action_controller' do
> > +      ActiveSupport.on_load(:action_controller) do
> > +        include(::ActionController::Serialization)
> > +      end
> > +    end
> > +
> > +    # This hook is run after the action_controller railtie has set the configuration
> > +    # based on the *environment* configuration and before any config/initializers are run
> > +    # and also before eager_loading (if enabled).
> > +    initializer 'active_model_serializers.set_configs', :after => 'action_controller.set_configs' do
> > +      ActiveModelSerializers.logger = Rails.configuration.action_controller.logger
> > +      ActiveModelSerializers.config.cache_store     = Rails.configuration.action_controller.cache_store
> > +      ActiveModelSerializers.config.perform_caching = Rails.configuration.action_controller.perform_caching
> Related to this... It seems like it would be better to not copy the ActiveModelSerializers.config.cache_store to _cache on the serializers themselves. That creates a situation where the individual serializer's cache store is dependent on the order of configuration files, serializers getting loaded, and lazy-loading of action_controller. That is outside the scope of this PR but something that could be considered. 
> 
> —
> Reply to this email directly or view it on GitHub.
>